<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-O5SIUU5" name="POM.ParallelTesting" tests="14" failures="0" timestamp="08 May 2018 05:00:54 GMT" time="18.019" errors="3">
  <testcase name="setup_server" time="0.076" classname="POM.ParallelTesting">
    <error type="org.testng.TestNGException" message="
Parameter &#039;deviceName&#039; is required by @Test on method setup_server but has not been marked @Optional or defined
in C:\Users\Acer\AppData\Local\Temp\testng-eclipse-1693463192\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'deviceName' is required by @Test on method setup_server but has not been marked @Optional or defined
in C:\Users\Acer\AppData\Local\Temp\testng-eclipse-1693463192\testng-customsuite.xml
	at org.testng.internal.Parameters.createParams(Parameters.java:248)
	at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:286)
	at org.testng.internal.Parameters.createParameters(Parameters.java:547)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:672)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1316)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1044)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1134)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup_server -->
  <testcase name="LoginTestCase" time="0.074" classname="POM.ParallelTesting">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at POM.Login_Page.clickclose(Login_Page.java:103)
	at POM.ParallelTesting.LoginTestCase(ParallelTesting.java:135)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- LoginTestCase -->
  <testcase name="JobsFeedTestCase" time="0.068" classname="POM.ParallelTesting"/>
  <testcase name="ApplicationTestCase" time="0.126" classname="POM.ParallelTesting"/>
  <testcase name="InterviewTestCase" time="0.108" classname="POM.ParallelTesting"/>
  <testcase name="OfferTestCase" time="0.196" classname="POM.ParallelTesting"/>
  <testcase name="JobSearchLocationTestCase" time="2.782" classname="POM.ParallelTesting"/>
  <testcase name="JobSearchTestCase" time="2.075" classname="POM.ParallelTesting"/>
  <testcase name="JobSearchpPayTestCase" time="2.146" classname="POM.ParallelTesting"/>
  <testcase name="TrainingTestCase" time="4.087" classname="POM.ParallelTesting"/>
  <testcase name="MytTrainingTestCase" time="2.077" classname="POM.ParallelTesting"/>
  <testcase name="WalletTestCase" time="2.134" classname="POM.ParallelTesting"/>
  <testcase name="SettingsTestCase" time="2.068" classname="POM.ParallelTesting"/>
  <testcase name="ForgetTestCase" time="0.002" classname="POM.ParallelTesting">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at POM.Forget_Page.clickclose(Forget_Page.java:65)
	at POM.ParallelTesting.ForgetTestCase(ParallelTesting.java:214)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- ForgetTestCase -->
</testsuite> <!-- POM.ParallelTesting -->
